### ISSUE TITLE
Road Map

### ISSUE THREAD

**Issue by Redjumpman** [Redjumpman/Jumper-Cogs#76]
Join the conversation on my github to comment and request features.

With Red V3 approaching closer each day I want to limit how much work I do on Red V2 cogs. Adding more content now, means a longer port time in the future. I do still plan to support V2 cogs through bug patches, so please continue to submit issues. When V3 has a full release I will discontinue support.

Currently, I do not have a V3 develop branch active. This is because I am still waiting on some finishing touches for Economy to be completed. If you want to learn more about the development of Economy check it out [here](https://github.com/palmtree5/Red-DiscordBot/tree/v3-economy/cogs/economy) 

Like all previous road maps these tasks are subject to be removed or updated based on the development process.

# Shop

- [ ] Complete rewrite of the code
- [ ] Bulk item uploader. Possibly through txt or csv files.
- [ ] Restock function. Automatically restocks items
- [ ] Optional embed outputs #60 
- [ ] Optional global mode to allow a shop for all your connected servers #72
- [ ] Notification messages when an admin grants redeem requests #44 
- [ ] Random Item feature #58 
- [ ] More customization buy msgs
- [ ] Allow users to sale back items or sell to other players #77
- [ ] Overhaul pending system

# Casino

- [ ] Multi-language support (work has already begun!)
- [ ] Better input for timers (H:M:S) instead of knowing the seconds
- [ ] New game
- [ ] ~Optional fees for transfers #29~ Casino chips will be removed in favor of credits
- [ ] Better way to check all the memberships
- [ ] Membership editor
- [ ] Whitelist / Blacklist
- [ ]  Overhaul patching system
- [x] Deprecate casino membership transfers
- [ ] Improved logging
- [ ] Bonus multipliers via memberships #97 

# Pokedex

- [ ] Switch to API or CSV

# Heist

- [ ] Items to affect outcomes
- [ ] Extended mode, repeats outcomes for really large player bases
- [ ] Code rework
- [ ] Solo heists
- [ ] Prison breaks

# Cookie

- [ ] Random events
- [ ] Pet store that sells items affecting the game
- [ ] Guard dogs
- [ ] Integration to use cookies for other cogs

**Comment by skeith** [comment]
Redjumpman, may I add?
Please add User ID on Pending list for shop. The list save their last name, but when changed, I find it hard to find who really purchase it. I'd also suggest to allow `pending showall` have filter to only show the `pending` stuff

**Comment by Redjumpman** [comment]
Added "Overhaul Pending System" It needs a lot of TLC. Those are good ideas.

**Comment by skeith** [comment]
I am also want to request a change on heist police timer. Instead of a fixed timer, make a random timer based on min and max timer

**Comment by Redjumpman** [comment]
Closing this in favor of using cards via Github Projects. You can find the new information located here: https://github.com/Redjumpman/Jumper-Cogs/projects
